admin/app/controllers/solidus_admin/addresses_controller.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module SolidusAdmin
+  class AddressesController < BaseController
+    include Spree::Core::ControllerHelpers::StrongParameters
+
+    before_action :load_order
+    before_action :validate_address_type
+
+    def new
+      address = @order.send("#{address_type}_address")
+      @order.send("build_#{address_type}_address", country_id: default_country_id) if address.nil?
+      address ||= @order.send("#{address_type}_address")
+      address.country_id ||= default_country_id if address.country.nil?
+
+      respond_to do |format|
+      end
+    end
+
+    def update
+      if @order.contents.update_cart(order_params)
+        flash[:notice] = t('.success')
+        redirect_to edit_order_url(@order)
+      else
+        flash.now[:error] = @order.errors.full_messages.join(", ")
```

Comment:
should we should show only errors defined on `:base`? 
(others being shown inline on fields)